sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -55,32 +60,53 @@ export class AssessmentAESTController extends BaseController {
    */
   @Get("application/:applicationId/requests")
   async getRequestedAssessmentSummary(
-    @Param("applicationId") applicationId: number,
+    @Param("applicationId", ParseIntPipe) applicationId: number,
   ): Promise<RequestAssessmentSummaryAPIOutDTO[]> {
+    let requestAssessmentSummary: RequestAssessmentSummaryAPIOutDTO;
+    const offeringChange =
+      await this.educationProgramOfferingService.getOfferingRequestsByApplicationId(
+        applicationId,
+      );
+    if (offeringChange) {
+      requestAssessmentSummary = {
+        id: offeringChange.id,
+        submittedDate: offeringChange.submittedDate,
+        status: OfferingStatus.ChangeAwaitingApproval,
```

Comment:
Please assign this from the offeringChange(offeringChange.offeringStatus) value instead of hard-code.